config/config.json.sample-ocis

```diff
@@ -10,9 +10,10 @@
     "scope": "openid profile email"
   },
   "apps": [
+    "external",
```

Comment:
As far as I remember the first app in the list is the "default" app that gets loaded when you open oCIS. As much as I like the idea of sorting apps in our sample configs, it might be necessary to put `files` first. Could you please double check this?

Reply:
how should an administrator know?

Reply:
Well, I didn't say that the solution is good. :-P But if we promote it in the example configs it gets even worse. Needs a fix asap if that issue is still present. Or as a (bad) workaround a hint in the docs.

Reply:
Tracked in https://github.com/owncloud/ocis/issues/2568